packages/core/test/render3/node_selector_matcher_spec.ts

```diff
@@ -81,10 +81,13 @@ describe('css selector matching', () => {
         ])).toBeFalsy(`Selector '[other]' should NOT match <span title="">'`);
       });
 
-      it('should match namespaced attributes', () => {
+      // TODO: this case will not work, need more discussion
+      // https://github.com/angular/angular/pull/34625/files#r401853803
```

Comment:
This link does not work for me? Maybe try:

```suggestion
      // https://github.com/angular/angular/pull/34625#discussion_r401791275
```

Reply:
Got it, thanks, updated.